packages/create-webpack-app/README.md

```diff
@@ -0,0 +1,11 @@
+# `create-webpack-app`
+
+> TODO: description
+
+## Usage
+
+```js
+const createWebpackApp = require("create-webpack-app");
+
+// TODO: DEMONSTRATE API
```

Comment:
Do we need to promote the API usage? IMO, CWA interaction should be limited to CLI.

Reply:
I was thinking the same thing, will surely work on a better readme.